README.md

```diff
@@ -56,99 +58,79 @@ $ pod install
+To Register an app:
 
-To create an app,  
-1. Click the **New registration** button on the top left of the page.
-2. On the app registration page,
-    - Name your app
-    - Under **Supported account types**, select **Accounts in any organizational directory and personal Microsoft accounts**
-    - Select **Register** to finish.
-3. After the app is created, you'll land on your app management page. Click **Authentication**, and add new Redirect URI with type **Public client (mobile & desktop)**. Enter redirect URI in format: `msauth.<app.bundle.id>://auth`. Replace <app.bundle.id> with the **Bundle Identifier** for your application.
-4. Hit the **Save** button in the top left, to save these updates. 
+1. Sign in to the [Azure portal](https://portal.azure.com) using either a work or school account.
+2. In the left-hand navigation pane, select the **Azure Active Directory** blade, and then select **App registrations**.
+3. Click on the **New registration** button at the top left of the page.
+4. On the app registration page,
+   - Name your app
+   - Under **Supported account types**, select **Accounts in any organizational directory (Any Azure AD directory - Multitenant) and personal Microsoft accounts (e.g. Skype, Xbox)**
+   - Click **Register** to finish.
+5. After the app is created, you'll land on your app management page. Take note of the **Application (client) ID** as this would be needed for the step 1C below.
+6. Click **Authentication**, and add new Redirect URI with type **Public client (mobile & desktop)**. Enter redirect URI in format: `msauth.<app_bundle_id>://auth`. Replace <app_bundle_id> with the **Bundle Identifier** for your application. 
+7. Hit the **Save** button in the top left, to save these updates.
 
-## 1B: Configure your application
+## 2B: Configure your application
+
+1. Update your application's redirect URI scheme in the  `Info.plist` file by replacing `msauth.com.microsoft.identitysample.MSALiOS` . Redirect URI scheme follows the format `msauth.[app_bundle_id]`. Make sure to substitue [app_bundle_id] with the **Bundle Identifier** for your application. 
 
-1. Add your application's redirect URI scheme to added in the portal to your `info.plist` file. It will be in the format of `msauth.<app.bundle.id>`
 ```xml
-    <key>CFBundleURLTypes</key>
+<key>CFBundleURLTypes</key>
+<array>
+  <dict>
+    <key>CFBundleURLSchemes</key>
     <array>
-        <dict>
-            <key>CFBundleTypeRole</key>
-            <string>Editor</string>
-            <key>CFBundleURLName</key>
-            <string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
-            <key>CFBundleURLSchemes</key>
-            <array>
-                <string>msauth.<app.bundle.id></string>
-            </array>
-        </dict>
+      <string>msauth.com.microsoft.identitysample.MSALiOS</string>
```

Comment:
should this be msauth.<app.bundle.id>?

Reply:
This is based on the instruction above.

Reply:
Instructions above suggest replacing this value with msauth.[app_bundle_id]. We are showing this value in all Info.plist samples and quoickstarts. I'd recommend to follow the same pattern here.

Reply:
Done.